jsonschema/cli.py

```diff
@@ -70,18 +70,25 @@ def validation_success(self, **kwargs):
 @attr.s
 class _PrettyFormatter(object):
 
-    _ERROR_MSG = dedent(
-        """\
-        ===[{type}]===({path})===
+    _WIDTH = 79
 
-        {body}
-        -----------------------------
-        """,
-    )
-    _SUCCESS_MSG = "===[SUCCESS]===({path})===\n"
+    @classmethod
+    def _json_formatter(cls, x):
+        return json.dumps(x, separators=(',\n', ': '), sort_keys=True)
+
+    def _simple_msg(self, path, type, header=False):
+        begin_end_chars = u'╒╕' if header is True else u'══'
+        return u'{}══[{}]═══({})'.format(begin_end_chars[0], type, path) \
+            .ljust(self._WIDTH - 1, u'═') + begin_end_chars[1]
```

Comment:
What I changed was:

```suggestion
        return u'{}══[{}]═══({}){}'.format(begin_end_chars[0], type, path, u'═' * self._WIDTH)[:self._WIDTH - 1] \
            + begin_end_chars[1]
```

That resolved the errors when I run the CLI.  However, the output gets mangled by some other part of pypy2:

```text
ââ  â  [ValidationError]â  â  â  (<stdin>)â  â  â  â  â  â  â  â  â  â  â  â   
7 is less than the minimum of 9

Failed validating u'minimum' in schema:
    {"minimum": 9,
    "multipleOf": 3,
    "not": {"const": 7}}

On instance:
    7
â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â  â
```

I think this is specific to pypy2 because the code works without that change under CPython 2.7.

I haven't gotten the test runner to work under pypy2 yet.

I'm beginning to wonder whether this shouldn't be considered a known issue with pypy2.  The pretty output wouldn't be recommended for that interpreter and those tests would be skipped, too.

Reply:
Ha!  This may be resolved by adding a `# -*- coding: UTF-8 -*-` line to the beginning of the file.  Checking on it…

Reply:
Adding a "coding" declaration for the file resolves the problem on my computer when I run the CLI with the 2.* versions of PyPy and CPython.  I still don't have the tests working in my 2.* environments.  I probably need to add that declaration to the CLI tests file, too.